CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Service/network certificate validity period is no longer hardcoded and can instead be set by operators and renewed by members:
+  - The new `network_certificate_initial_validity_days` (defaults to 1 day) configuration entry lets operators set the initial validity period for the network certificate (valid from the current system time).
```

Comment:
1 day seems like a very short default, I think we should default to 365 days instead.

Reply:
The logic behind this is the same as for the node certificate: a short initial validity period (1 day) forces operators to renew the certificate early (and hopefully automate the process!) and avoid learning about it in a year's time when the certificate unexpectedly expires. Is there any specific reason  you think this logic shouldn't apply to the service certificate as well?